server/api/user_registration/index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const express = require('express');
+const controller = require('./project.controller');
```

Comment:
This should import something like:

```js
const controller = require('./user_registration.controller');
```

Just fix that issue and we'll merge it in!